bls/src/test/java/tech/pegasys/artemis/util/bls/BLSPublicKeyTest.java

```diff
@@ -83,6 +84,18 @@ void succeedsWhenEqualsReturnsFalseForDifferentPublicKeys() {
     assertNotEquals(publicKey1, publicKey2);
   }
 
+  @Test
+  public void succeedsWhenEqualsReturnsTrueForEquivalentPublicKeysCreatedFromDifferentRawBytes() {
```

Comment:
nit; `final` modifier for the `BLSPublicKey`s

Reply:
Removed the finals that were there to be consistent within the file.  Being consistent with final everywhere is another of those highly-conflict-causing jobs we'll need to bite off at some point, but not today.